Lesson2_Authentication/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Lesson2_Authentication
+{
+    class Program
+    {
+        public static void Main()
+        {
+            Dictionary<string, string> users = new Dictionary<string, string>
+            {
+                { "user1", "password1" },
+                { "user2", "password2" },
+                { "user3", "password3" }
+            };
+            Console.WriteLine("Please enter your login");
+            string login = Console.ReadLine();
+
+            if (users.ContainsKey(login))
+            {
+                string password = users.GetValueOrDefault(login);
```

Comment:
Feature of the .NET 5 and .NET core last versions, better to not use such methods, but since i hadn't specified platform versions it is ok